src/main/resources/config.yml

```diff
@@ -28,6 +28,7 @@ researches:
   free-in-creative-mode: true
   enable-fireworks: true
   disable-learning-animation: false
+  dont-need-research-to-use: false
```

Comment:
This wording isn’t very straight forward, perhaps require-research-on-use and default this to true instead?

Reply:
definitely default the behavior to what it currently is

Reply:
That makes sense, easy change. (Just as a note to svr333 it was set to the default behavior to what it currently is he is just suggesting a name change and fitting value with the new name but still keep it to the default behavior to what it currently).